routes/articles.js

```diff
@@ -0,0 +1,118 @@
+const express = require('express');
+const router = express.Router();
+const middleware = require('../middleware/validate');
+const knex = require('../db');
+
+/************************
+ *  GET
+************************/
+
+router.get('/', (req, res) => {
+  const articles = knex('articles');
```

Comment:
You could merge this line and the line below into one line since you don't use the articles variable anywhere else in this function.